src/app/lib/utilities/preprocessor/rules/filterStraplines.js

```diff
@@ -0,0 +1,26 @@
+import pathOr from 'ramda/src/pathOr';
+
+const filterStraplines = data => {
```

Comment:
```suggestion
const filterGroupsWithoutStraplines = data => {
```

Reply:
Could you also rename this file to match the new name for the function?